app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -150,6 +151,17 @@ object TelemetryWrapper {
         val AVERAGE = "average"
     }
 
+
+    enum class BrowserContextMenuValue {
+        Link, Image, ImageWithLink;
+
+        override fun toString(): String = when(this) {
```

Comment:
\[ktlint\] WARNING: \[keyword\-spacing\] Missing spacing after "when" in column 47